PokeDemo/Models/Pokemon.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Pokemon.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+
+struct Pokemon: Decodable, Equatable {
```

Comment:
why Equatable?

Reply:
Removed.

Reply:
Then move it in unit test only.

Reply:
Move it in unit test target

Reply:
Covered in dc6c3b12b54f65c2219d3117fb365e29511a26ee